app/assets/stylesheets/theme_ursus/pages/_ur-static-stgallpage.scss

```diff
@@ -141,53 +145,135 @@
   gap: 16px;
   list-style-type: none;
 
-  li {
+  > li {
     display: flex;
     flex-direction: column;
     gap: 4px;
   }
 }
+.bibliography-links {
+  display: inline-flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  gap: 12px;
 
+  > li {
+    display: inline;
+  }
+
+  a {
+    padding: 1rem 0;
+    font-size: 10px;
+    font-weight: 600;
+    color: $ucla-blue;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+
+    &::after {
+      content: "";
+      width: 11px;
+      height: 11px;
+      margin-left: 4px;
+      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='16' fill='currentColor' viewBox='0 0 16 16'%3E%3Cpath fill-rule='evenodd' d='M8.636 3.5a.5.5 0 0 0-.5-.5H1.5A1.5 1.5 0 0 0 0 4.5v10A1.5 1.5 0 0 0 1.5 16h10a1.5 1.5 0 0 0 1.5-1.5V7.864a.5.5 0 0 0-1 0V14.5a.5.5 0 0 1-.5.5h-10a.5.5 0 0 1-.5-.5v-10a.5.5 0 0 1 .5-.5h6.636a.5.5 0 0 0 .5-.5z'/%3E%3Cpath fill-rule='evenodd' d='M16 .5a.5.5 0 0 0-.5-.5h-5a.5.5 0 0 0 0 1h3.793L6.146 9.146a.5.5 0 1 0 .708.708L15 1.707V5.5a.5.5 0 0 0 1 0v-5z'/%3E%3C/svg%3E");
+      background-position: center;
+      background-repeat: no-repeat;
+      background-size: contain;
+      display: inline-block;
+    }
+  }
+}
 
 .stgall-sub-links {
   display: flex;
   flex-direction: row;
   justify-content: flex-start;
-  align-items: center;
-  gap: 8px;
+  align-items: flex-start;
+  gap: 2px;
+  position: relative;
 
   a,
   button {
     background-color: $ucla-lightest-blue-2;
-    color: $ucla-darkest-blue;
+    color: $ucla-darker-blue;
     line-height: 1;
     margin: 0.5rem 0 0.5rem;
     padding: 0.7rem 1rem;
-    border: none;
-  }
-}
+    border: 2px solid transparent;
 
-.stgall-work-item-content {}
+    display: inline-block;
+    position: relative;
+    z-index: 2;
+    top: 2px;
 
-.stgall-work-item-bibliography {}
-
-.stgall-work-item-transcription {}
-
-/*.manuscript-bibliography {
+    &:hover:not(.active) {
+      background-color: $ucla-blue;
+      color: $white;
+    }
 
+    &:focus,
```

Comment:
@pghorpade after the logic is put in place in the js, you can move 213 and to 208 so that the hover and focus states have the same styling.